stor/test.py

```diff
@@ -309,7 +316,9 @@ def setup_file(self, obj):
                              folder='/'+dx_p.parent.lstrip('/'),
                              project=self.proj_id) as f:
             f.write('data'.encode())
-        f.wait_on_close(20)  # to allow for max of 20s for file state to go to closed
+
+        should_mock = self.cassette.rewound
+        self.mock_time_sleep(should_mock, f.wait_on_close, 20)
```

Comment:
rather than pass the `should_mock` parameter (which looks a little goofy) how about making a `file_close_wait_time` property on the class that looks like:

```
def file_close_wait_time(self):
    if self.cassette.rewound:
        # do not waste time waiting when we are not making network requests
        return 0
    else:
        return 20
```

and then switch to

```
f.wait_on_close(self.file_close_wait_time)
```